autoload/nvimdev.vim

```diff
@@ -136,6 +145,16 @@ function! nvimdev#init(path) abort
       execute 'autocmd BufRead ' . s:path . '/build/* setlocal readonly nomodifiable'
       execute 'autocmd BufRead ' . s:path . '/.deps/* setlocal readonly nomodifiable'
     endif
+
+    " Setup/configure projectionist.
+    if get(g:, 'loaded_projectionist', 0)
+      autocmd User ProjectionistDetect call nvimdev#setup_projectionist(g:projectionist_file)
+
+      " Init for first buffer, since this is called on BufEnter itself.
+      call ProjectionistDetect(expand('%:p'))
+
+      command! NvimDiff exe 'diffsplit' projectionist#query_file('alternate')[0]
```

Comment:
TODO: better name?

Should/could also set `:set diffopt+=iwhite` (but it is a global setting currently).